### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by c4pt0r**
make document more legible.

### CODE REVIEW

**Comment by c4pt0r** [comment]
@shenli @siddontang PTAL

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ TiDB is a distributed SQL database. Inspired by the design of Google F1, TiDB su
 ## Operating TiDB clusters
  - [Deployment overview](op-guide/overview.md)
  - [Supported platform](op-guide/build.md#supported-platforms)
- - [Building the TiDB components](op-guide/build.md)
  - [Configuration](op-guide/configuration.md)
  - [Setting up clusters](op-guide/clustering.md)
  - [Running the TiDB clusters inside containers](op-guide/docker.md)
 
 ## Developing with TiDB
+ - [Building requirements](op-guide/build.md)
```
Comment (shenli) [inline README.md]: Should we move the file to dev-guide dir?

Comment (c4pt0r) [inline README.md]: I think so.

Comment (c4pt0r) [inline README.md]: I'll do it in this PR.

Comment (c4pt0r) [inline README.md]: DONE

**Comment by shenli** [comment]
LGTM
@siddontang PTAL

**Comment by shenli** [comment]
@siddontang PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**